src/index.ts

```diff
@@ -19,7 +21,9 @@ app.use(errorHandlingMiddleware);
 const start = async () => {
     try {
         await sequelize.authenticate();
-        await sequelize.sync();
+        await User.sync();
+        await Token.sync();
+        await Profile.sync();
```

Comment:
in the future u will have a lot of models, so better to move them to another func or file